release/rllib_tests/4gpus_64cpus.yaml

```diff
@@ -1,7 +1,7 @@
 cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
 region: us-west-2
 
-max_workers: 0
+max_workers: 5
```

Comment:
Are these changes intentional? It seems like only auto_scale.yaml is needed for the test.

Reply:
ah yes. I switched around. I actually used all of these before settling on a new autoscale.yaml config.
these configs don't work without the changes. max_workers==0 and no attached disk, etc.
they are not used right now. so I am fine with keeping them as is, or simply deleting them, whichever you prefer.